LockboxXCUITests/LockboxXCUITests.swift

```diff
@@ -75,25 +75,27 @@ class LockboxXCUITests: BaseTestCase {
             let regexpUid = "(uid=[a-z0-9]{0,32}$?)"
             let regexCode = "(code=[a-z0-9]{0,32}$?)"
             if let rangeUid = responseString?.range(of:regexpUid, options: .regularExpression) {
-                uid = (responseString?.substring(with:rangeUid))!
+                uid = (responseString?.substring(with:rangeUid))
             }
 
             if let rangeCode = responseString?.range(of:regexCode, options: .regularExpression) {
-                code = (responseString?.substring(with:rangeCode))!
+                code = (responseString?.substring(with:rangeCode))
             }
-
-            let finalCodeIndex = code.index(code.endIndex, offsetBy: -32)
-            let codeNumber = code[finalCodeIndex...]
-            let finalUidIndex = uid.index(uid.endIndex, offsetBy: -32)
-            let uidNumber = uid[finalUidIndex...]
-            self.completeVerification(uid: String(uidNumber), code: String(codeNumber)) {
-                done()
+            if (code != nil && uid != nil) {
+                let finalCodeIndex = code.index(code.endIndex, offsetBy: -32)
+                let codeNumber = code[finalCodeIndex...]
+                let finalUidIndex = uid.index(uid.endIndex, offsetBy: -32)
+                let uidNumber = uid[finalUidIndex...]
+
+                self.completeVerification(uid: String(uidNumber), code: String(codeNumber)) {
+                    done()
+                }
```

Comment:
This looks complicated; can you simplify the logic by using `guard` statements?

Reply:
👍 I will try to simplify it.
Thanks!